.changeset/little-clocks-invite.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': patch
+'@itwin/itwinui-css': patch
+---
+
+Fixed Dialog blurring and resizing issues by removing `overflow: hidden` and inline `transform` styles.
```

Comment:
Should we mention `Modal` also?

Also, is `overflow: hidden` and inline `transform` styles internal information that is impertinent for users, and thus should not be in the changeset?

Reply:
Mentioned Modal [`8a16169`](https://github.com/iTwin/iTwinUI/pull/1795/commits/8a161697259b493a725af650a00e9e1ead5e1923)

These styles are internal details yes, but as you can see in the linked issues, our users are already messing with such things, so it's important to mention. If they were manually adding `overflow: visible`, they can remove it after this release.